src/components/stepforms/schemas/evolution.ts

```diff
@@ -0,0 +1,70 @@
+import { addNotInColumnNamesConstraint, StepFormType } from './utils';
+
+const createSchema = () => ({
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Evolution step',
+  type: 'object',
+  properties: {
+    name: {
+      type: 'string',
+      consnt: 'evolution',
+    },
+    dateCol: {
+      type: 'string',
+      minLength: 1,
+      title: 'Date column',
+      description: 'Date column used to compute evolution',
+      attrs: {
+        placeholder: 'Select a column',
+      },
+    },
+    valueCol: {
+      type: 'string',
+      minLength: 1,
+      title: 'Value column',
+      description: 'The value column',
+      attrs: {
+        placeholder: 'Select a column',
+      },
+    },
+    evolutionType: {
+      type: 'string',
+      title: 'Compute evolution versus',
+      enum: ['vsLastYear', 'vsLastMonth', 'vsLastWeek', 'vsLastDay'],
+    },
+    uniqueIndexCols: {
+      type: 'array',
+      minItems: 0,
+      items: {
+        type: 'string',
+        minLength: 1,
+      },
+      title: 'uniqueIndexCols',
+      description: 'Index columns',
```

Comment:
I am not sure of the naming: `uniqueIndexCols`. If we display "Index columns" why do we use `uniqueIndexCols` ? And also, please do not use abbreviation, here 'Col'. 
For these reasons I suggest to use `IndexColumns` instead of `uniqueIndexCols`.




disapprove
--